bridge/engine/SsrcWhitelist.h

```diff
@@ -8,6 +8,15 @@ namespace bridge
 
 struct SsrcWhitelist
 {
+    SsrcWhitelist() : enabled(false), numSsrcs(0), ssrcs{0} {}
+    SsrcWhitelist& operator=(const SsrcWhitelist& o)
```

Comment:
I guess we shouldn't specify a copy/move assign/ctor when it is not change the default behaviour. Besides being unnecessary code, this class has all requirements to be trivially copiable but this explicit copy assignment make it non-trivial and some compiler's optimisations opportunities are lost